packages/core/src/VRange.ts

```diff
@@ -298,6 +323,7 @@ export class VRange {
         } else {
             reference.before(this.start);
         }
+        this.editor.selection.range.modifiers = undefined;
```

Comment:
Ca c'est louche hein, ca devrait être `this.modifiers = undefined` il me semble, et pareil dans l'autre fonction.